cpp/ql/test/query-tests/Security/CWE/CWE-119/semmle/tests/OverflowStatic.expected

```diff
@@ -5,4 +5,5 @@
 | tests.cpp:245:42:245:42 | 6 | Potential buffer-overflow: 'global_array_5' has size 5 not 6. |
 | tests.cpp:349:2:349:14 | access to array | Potential buffer-overflow: 'charArray' has size 10 but 'charArray[10]' may be accessed here. |
 | tests.cpp:350:17:350:29 | access to array | Potential buffer-overflow: 'charArray' has size 10 but 'charArray[10]' may be accessed here. |
+| tests.cpp:608:33:608:43 | access to array | Potential buffer-overflow: 'buffer' has size 100 but 'buffer[101]' may be accessed here. |
```

Comment:
Is this still expected?

Reply:
Hmmm, wait. No, it's not 🤔. Looks like I was too fast there. Let me look into it!

Reply:
Fixed in https://github.com/github/codeql/pull/12611/commits/2ce0d2b7ee6482209dbd07d9f1bf8f9da1170e26. I had just forgotten to accept test changes in that file 😂.